units/host_allocator.go

```diff
@@ -288,18 +288,29 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	span := trace.SpanFromContext(ctx)
 	span.SetAttributes(
 		attribute.String(evergreen.DistroIDOtelAttribute, distro.Id),
+		attribute.String(fmt.Sprintf("%s.distro_provider", hostAllocatorAttributePrefix), distro.Provider),
+		attribute.String(fmt.Sprintf("%s.total_runtime", hostAllocatorAttributePrefix), distroQueueInfo.ExpectedDuration.String()),
+		attribute.String(fmt.Sprintf("%s.time_to_empty", hostAllocatorAttributePrefix), timeToEmpty.String()),
+		attribute.String(fmt.Sprintf("%s.time_to_empty_no_spawns", hostAllocatorAttributePrefix), timeToEmptyNoSpawns.String()),
```

Comment:
It seems like these are duplicates of `runtime_secs`, `time_to_empty_mins`, and `time_to_empty_no_spawn_mins`. Can we remove these ones? They seem a little less convenient to use since they're strings rather than numbers.